pyuvsim/tests/test_uvsim.py

```diff
@@ -65,7 +65,7 @@ def triangle_pos():
     hera_uv = UVData()
     hera_uv.read_uvfits(longbl_uvfits_file,
                         ant_str='cross')  # consists of a right triangle of baselines with w term
-    hera_uv.unphase_to_drift(use_ant_pos=True)
+    hera_uv.unproject_phase(use_ant_pos=True)
```

Comment:
was this just a direct method name change in pyuvdata?

Reply:
yes.